### PR TITLE
finish reformatting of `round`

### PR DESCRIPTION
**Pull request by ahmedo42**
#1988

### CODE REVIEW

**Comment by RickSanchezStoic** [comment]
Check and fix the lint errors as well as the test-docstrings / run-docstring-tests. They both should go green.

**Comment by ahmedo42** [comment]
@RickSanchezStoic it seems to me that test-docstrings are failing for reasons unrelated to this PR -precisely `repeat()`- and lint tests are already green, can you confirm? or am I missing something?

**Comment by RickSanchezStoic** [comment]
> @RickSanchezStoic it seems to me that test-docstrings are failing for reasons unrelated to this PR -precisely `repeat()`- and lint tests are already green, can you confirm? or am I missing something?

It seems like your fork is not updated, first try adding examples with `out` argument and then try updating your fork.

**Comment by ahmedo42** [comment]
@RickSanchezStoic  I think it's fixed now and examples with `out` are already there

**Comment by ahmedo42** [comment]
@RickSanchezStoic where is the review? can you request changes in the code or be more specific? , I went through the deep dive extensively and I don't understand what's still missing 

Again, examples with the `out ` argument  are already implemented so I didn't need to add them

**Review by RickSanchezStoic — CHANGES_REQUESTED** [review]

**Comment by RickSanchezStoic** [comment]
My bad @ahmedo42, it should be visible now.

File: ivy/array/elementwise.py [hunk 1]
```diff
@@ -1332,23 +1332,34 @@ def round(self: ivy.Array, *, out: Optional[ivy.Array] = None) -> ivy.Array:
...
+        >>> print(y)
+        ivy.array([-94., 256., 0., -0., 37.])
+
+        >>> x = ivy.array([[ -1., -67.,  0.,  15.5,  1.], [3, -45, 24.7, -678.5, 32.8]])
         >>> y = x.round()
         >>> print(y)
-        ivy.array([1., 2., 4.])
+        ivy.array([[-1., -67., 0., 16., 1.],
+        [3., -45., 25., -678., 33.]])
         """
```
[the remark this answers is not in the document]
Comment (ahmedo42) [reply ivy/array/elementwise.py:1442]: https://github.com/unifyai/ivy/blob/2539d99260dfe85c2ec97d1124e407e841196da9/ivy/functional/ivy/elementwise.py#L3660
it's in the functional module, should It also be implemented in the array module?

[the remark this answers is not in the document]
Comment (RickSanchezStoic) [reply ivy/array/elementwise.py:1442]: We should add it for the array module too.

**Review by RickSanchezStoic — CHANGES_REQUESTED** [review]

File: ivy/array/elementwise.py [hunk 2]
```diff
@@ -1411,23 +1411,39 @@ def round(self: ivy.Array, *, out: Optional[ivy.Array] = None) -> ivy.Array:
...
+        ivy.array([ 6., -8.,  0., -4.,  7.])
+
+        >>> x = ivy.array([-94.2, 256.0, 0.0001, -5.5, 36.6])
+        >>> y = x.round()
+        >>> print(y)
+        ivy.array([-94., 256., 0., -6., 37.])
+
+        >>> x = ivy.array([0.23, 3., -1.2])
+        >>> y = ivy.zeros(3)
+        >>> print(x.round(out=y))
```
Comment (RickSanchezStoic) [inline ivy/array/elementwise.py:1439]: Since `y` is passed here to the `out` argument, it should be `print(y)` after `x.round(out=y)`

**Review by RickSanchezStoic — CHANGES_REQUESTED** [review]

**Comment by RickSanchezStoic** [comment]
lgtm! closes #1988

**Comment by RickSanchezStoic** [comment]
Closes #1988

**Review by RickSanchezStoic — APPROVED** [review]

**Pull request merged**